### PR TITLE
[Merged by Bors] - chore: bump futures-aio via k8-client

### PR DESCRIPTION
**Pull request by pinkforest**
Related to
https://github.com/infinyon/fluvio/pull/2232

### Dependant on 
fluvio-future bump to 0.3.14 under;
- https://github.com/infinyon/future-aio/pull/134

### Publish

This PR bumps k8-client to 0.5.1

Which in turn enables bump in fluvio:

https://github.com/infinyon/fluvio/pull/2232

### CODE REVIEW

**Review by sehz — CHANGES_REQUESTED** [review]

File: src/k8-metadata-client/Cargo.toml [hunk 1]
```diff
@@ -15,5 +15,5 @@ serde = { version ="1.0.103", features = ['derive'] }
 serde_json = "1.0.40"
 async-trait = "0.1.21"
 k8-diff = { version = "0.1.0", path = "../k8-diff"}
-k8-types = { version = "0.5.0", path = "../k8-types" }
+k8-types = { version = "0.5.2", path = "../k8-types" }
```
[the remark this answers is not in the document]
Comment (sehz) [reply src/k8-metadata-client/Cargo.toml:18]: Then need to bump up the cargo version

[the remark this answers is not in the document]
Comment (wendyOn) [reply src/k8-metadata-client/Cargo.toml:18]: No need as k8-types is already published as 0.5.2 but the internal reference that is ignored with path = "" was just updated internally. k8-client is the only thing that needs publish.

File: src/k8-client/Cargo.toml [hunk 2]
```diff
@@ -33,8 +33,8 @@ serde = { version="1.0.108", features=['derive'] }
 serde_json = "1.0.40"
 serde_qs = "0.8.0"
 async-trait = "0.1.42"
-fluvio-future = { version="0.3.5", features=["net", "task"] }
-k8-metadata-client = { version="3.2.0", path="../k8-metadata-client" }
+fluvio-future = { version="0.3.14", features=["net", "task"] }
+k8-metadata-client = { version="3.3.1", path="../k8-metadata-client" }
```
Comment (sehz) [inline src/k8-client/Cargo.toml:37]: no need to bump up this

Comment (wendyOn) [reply src/k8-client/Cargo.toml]: I rolled it back to 3.2.0

**Review by sehz — APPROVED** [review]
LGTM

**Pull request closed without merging**